src/use-hotkeys/use-hotkeys.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller } from 'stimulus'
+// @ts-ignore
+import hotkeys from 'hotkeys-js'
+
+export const bindHotkeys = (controller: Controller) => {
```

Comment:
We might want to provide a second option to define whether or not to [filter](https://github.com/jaywcjlove/hotkeys#filter)